marklogic-data-hub-central/ui/e2e/cypress/support/pages/login.tsx

```diff
@@ -18,6 +18,10 @@ class LoginPage {
     cy.wait(2000);
     //cy.window().its("stompClientConnected").should("exist");
   }
+
+  clickTitle() {
+    cy.get(".Overview_title__2eqv4").click();
```

Comment:
I think we should avoid selecting elements like this because scss autogenerates the last part making it variable. This is prone to intermittent failures. :(